static/styles/settings.css

```diff
@@ -501,16 +501,22 @@ input[type="checkbox"] {
     word-break: break-word;
 }
 
+.account-heading {
+    margin-right: 10px;
+}
+
 .alert-notification {
     display: inline-block;
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    word-break: break-word;
```

Comment:
`break-word` is depreceated as mentioned in https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values.

Reply:
Have changed it to `break-word: normal` with `overflow-wrap: anywhere`